wifiphisher/common/opmode.py

```diff
@@ -107,6 +107,14 @@ def _check_args(self, args):
                 '] --nojamming (-nJ) and --jamminginterface (-jI)'
                 'cannot work together.')
 
+	if args.captiveportal and not args.internetinterface:
```

Comment:
This block here is not indented properly. Remember, indentation matters in Python! For example, with the current indentation the next if statement (`if args.lure10_exploit and args.nojamming`) will only be executed if the code passes your check.